public/wp-content/themes/coloredcow/src/scss/variables.scss

```
@@ -6,30 +6,30 @@ $theme-colors: (
     'theme-secondary': $theme-secondary,
 );
 
-$spacer: 10px;
+$spacer: 1rem;
 $spacers: (
     0: 0,
-    1: ($spacer * 0.25), // 2.5px
-    2: ($spacer * 0.5), // 5px
-    3: ($spacer * 1), // 10px
-    4: ($spacer * 1.5), // 15px
-    5: ($spacer * 2), // 20px
-    6: ($spacer * 2.5), // 25px
-    7: ($spacer * 3), //30px
-    8: ($spacer * 3.5), //35px
-    9: ($spacer * 4), //40px
-    10: ($spacer * 4.5), // 45px
-    11: ($spacer * 5), // 50px
-    12: ($spacer * 5.5), // 55px
-    13: ($spacer * 6), // 60px
-    14: ($spacer * 6.5), // 65px
-    15: ($spacer * 7), // 70px
-    16: ($spacer * 7.5), // 75px
-    17: ($spacer * 8), // 80px
-    18: ($spacer * 8.5), // 85px
-    19: ($spacer * 9), // 90px
-    20: ($spacer * 9.5), // 95px
-    21: ($spacer * 10), // 100px
+    1: ($spacer * 0.15625), // 2.5px
+    2: ($spacer * 0.3125), // 5px
```

I think you don't need to add complex decimals. You can simply do:
```suggestion
    1: ($spacer * (2.5/16)), // 2.5px
    2: ($spacer * (5/16)), // 5px
    // and so on
```

or the `math.div` version of the above.

It'll be easier to understand and copy also.

---

Modified the spacers for `math.div()` usage.